### PR TITLE
refactor(settings): refactored to ngrx 8

### PR DESCRIPTION
**Pull request by abedzantout**
## What:

Refactor settings ngrx functions to ngrx 8.

Issue number: https://github.com/tomastrajan/angular-ngrx-material-starter/issues/472

### CODE REVIEW

**Comment by tomastrajan** [comment]
Hi @abedzantout ! Thank you for your contribution, the CI build seems to be red, can you have a look ? Thank you!

**Comment by abedzantout** [comment]
@tomastrajan Done

File: projects/angular-ngrx-material-starter/src/app/core/settings/settings.actions.spec.ts [hunk 1]
```diff
@@ -1,90 +1,90 @@
...
-  ActionSettingsChangeTheme,
+  actionSettingsChangeAnimationsElements,
+  actionSettingsChangeAnimationsPage,
+  actionSettingsChangeAnimationsPageDisabled,
+  actionSettingsChangeAutoNightMode,
+  actionSettingsChangeHour,
+  actionSettingsChangeLanguage,
+  actionSettingsChangeStickyHeader,
+  actionSettingsChangeTheme,
   SettingsActionTypes
```
Comment (tomastrajan) [inline projects/angular-ngrx-material-starter/src/app/core/settings/settings.actions.spec.ts]: Hmm, this one ( `SettingsActionTypes` )was deleted in the file but is still imported here ?

Comment (timdeschryver) [reply projects/angular-ngrx-material-starter/src/app/core/settings/settings.actions.spec.ts]: I think it still has to be deleted - https://github.com/tomastrajan/angular-ngrx-material-starter/blob/a3bc534412b4f1397cac898d922706c0a88f4b15/projects/angular-ngrx-material-starter/src/app/core/settings/settings.actions.ts#L5

Comment (abedzantout) [reply projects/angular-ngrx-material-starter/src/app/core/settings/settings.actions.spec.ts]: Hmm, so you want to remove the SettingsActionTypes enum completely? We're using the same structure across almost all actions.

Comment (tomastrajan) [reply projects/angular-ngrx-material-starter/src/app/core/settings/settings.actions.spec.ts]: As far as I understand, with new action creators there is  no need to have actions union type anymore ? cc @timdeschryver

Comment (timdeschryver) [reply projects/angular-ngrx-material-starter/src/app/core/settings/settings.actions.spec.ts]: Yes, you can remove the enum completely.
Previously you had the enum to
 1) use it as the type to create the action
 2) use it in your reducers

Now, this becomes obsolete.

**Pull request merged**